beesdoo_pos/static/src/js/beesdoo.js

```diff
@@ -1,7 +1,7 @@
 odoo.define('beescoop.pos', function (require) {
     "use strict";
     var module   = require("point_of_sale.screens");
-    var Model = require('web.DataModel');
+    var rpc = require('web.rpc')
```

Comment:
I am unsure about this, but since we use `this._rpc`, `rpc` import can be removed